File: src/app/core/types/category.ts
export class Category {
  id: number;
  title: string;
  image: string;
  imageUrl: string;
  imageCrop: string;
  imageCropUrl: string;
  imageExtension: string;

  constructor(id: number, title: string, image: string, imageUrl: string, imageCrop: string,
    imageCropUrl: string, imageExtension: string) {
    this.id = id;
    this.title = title;
    this.image = image;
    this.imageUrl = imageUrl;
    this.imageCrop = imageCrop;
    this.imageCropUrl = imageCropUrl;
    this.imageExtension = imageExtension;
  }
}

File: backend/Marbomil.Infrastructure/Entities/Database/Category.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Marbomil.Infrastructure.Entities.Database
{
    public class Category : IDbEntity
    {
        public int Id { get; set; }

        [MaxLength(100)]
        public string Title { get; set; }

        [MaxLength(1000)]
        public string ImageUrl { get; set; }

        [MaxLength(1000)]
        public string ImageCropUrl { get; set; }

        public virtual ICollection<Product> Products { get; set; }
    }
}
